src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/QdmEntity.tsx

```diff
@@ -10,6 +10,7 @@ import _ from "lodash";
 export const Location_Attributes = ["LocationType"];
 export const Practitioner_Attributes = ["Role", "Specialty", "Qualification"];
 export const CarePartner_Attributes = ["Relationship"];
+export const Organization_Attributes = ["OrganizationType"];
```

Comment:
this casing seems a bit odd, neither capital nor camelCase.

Reply:
thank you Ashok, I have changed it